packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -63,6 +63,8 @@ export function defaultSerialize(
       object[name] = get(model, name);
     } else if (/_id$/.test(name)) {
       object[name] = get(model, 'id');
+    } else if (isProxy(model)) {
+      object[name] = get(model, name);
```

Comment:
I could also put this in the first if expression with `&&`.   Up to you if you feel having this final else if is better.  Or perhaps it doesn't matter 😄 .